Makefile

```diff
@@ -10,7 +10,6 @@ build: ## Create a new image
 setup: build ## Set up a clean database and node_modules folder for running the app or the specs in docker
 	docker-compose down -v
 	docker-compose run --rm web bundle exec rake db:setup
-	docker-compose run --rm web yarn install
```

Comment:
Is this step no longer necessary?

Reply:
The assets:precompile step in the Dockerfil triggers the install steps so this was was introducing an unnecessary delay doing it again here, so I removed it.

Reply:
I'm not sure why it shouldn't be — `node_modules` is a volume so on a freshly created setup it will be empty?

Reply:
The `yarn install` in the Dockerfile will bake `node_modules` into the container, and I agree this is what we want for an production image.

When the app is run via `docker-compose.yml` a docker volume is mounted for 
`node_modules`. (see the `volumes` key in that file). This is to support local development workflows where the app may be run outside Docker and the host's `node_modules` folder may clash with the one inside (see b1b28fc1).

I was under the impression that this meant starting the app from cold meant the `node_modules` volume was empty, but reading up, I didn't understand `docker-compose` volume behaviour. From the [docs](https://docs.docker.com/storage/volumes/):

> If you start a container which creates a new volume... and the container has files or directories in the directory to be mounted (such as /app/ above), the directory’s contents are copied into the volume. The container then mounts and uses the volume, and other containers which use the volume also have access to the pre-populated content.

**tl;dr** this change is :+1:

Reply:
We discussed this IRL and agreed that a stock `docker-compose.override.yml` holding dev-only config would clarify things somewhat. https://docs.docker.com/compose/extends/#adding-and-overriding-configuration. In this configuration we don't mount node_modules (or `/app` 🎉 ) during the build process in Azure

Reply:
Addressed as per discussion with @duncanjbrown in commit 86d4973.